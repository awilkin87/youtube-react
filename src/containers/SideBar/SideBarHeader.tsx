import { Menu } from 'semantic-ui-react';
import styles from './SideBarHeader.module.scss';

/** React props for SideBarHeader. */
export interface SideBarHeaderProps {
  title: string
}

/** Header for a subsection of the sidebar menu. */
const SideBarHeader: React.FunctionComponent<Readonly<SideBarHeaderProps>> = function (props) {
  return (
    <Menu.Item>
      <Menu.Header className={styles.sidebarHeader}>{props.title.toUpperCase()}</Menu.Header>
    </Menu.Item>
  );
}

export default SideBarHeader;