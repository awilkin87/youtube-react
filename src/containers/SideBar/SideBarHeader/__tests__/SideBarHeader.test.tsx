import { shallow } from "enzyme";
import SideBarHeader from '../SideBarHeader';

test('Renders SideBarHeader', () => {
  const wrapper = shallow(<SideBarHeader title='Header Title' />);
  expect(wrapper).toMatchSnapshot();
});