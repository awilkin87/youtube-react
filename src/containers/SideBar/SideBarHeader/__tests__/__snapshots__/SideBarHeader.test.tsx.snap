// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders SideBarHeader 1`] = `
<MenuItem>
  <MenuHeader
    className="sidebarHeader"
  >
    HEADER TITLE
  </MenuHeader>
</MenuItem>
`;
